commands/utils/params_test.go

```diff
@@ -169,6 +172,42 @@ func TestExtractAndAssertRepoParams(t *testing.T) {
 	}
 }
 
+func TestBuildRepoAggregatorWithEmptyScan(t *testing.T) {
+	SetEnvAndAssert(t, map[string]string{
+		JFrogUrlEnv:     "http://127.0.0.1:8081",
+		JFrogTokenEnv:   "token",
+		GitProvider:     string(GitHub),
+		GitRepoOwnerEnv: "jfrog",
+		GitRepoEnv:      "frogbot",
+		GitTokenEnv:     "123456789",
+	})
+	defer func() {
+		assert.NoError(t, SanitizeEnv())
+	}()
+	server, gitParams, err := extractClientServerParams()
+	assert.NoError(t, err)
+	configFileContent, err := ReadConfigFromFileSystem(configEmptyScanParamsTestFile)
+	assert.NoError(t, err)
+	configAggregator, err := BuildRepoAggregator(configFileContent, gitParams, server)
+	assert.NoError(t, err)
+	assert.Len(t, configAggregator, 1)
+	scan := configAggregator[0].Scan
+	assert.False(t, scan.IncludeAllVulnerabilities)
+	assert.False(t, scan.FixableOnly)
+	assert.Empty(t, scan.MinSeverity)
+	assert.Empty(t, scan.JfrogReleasesRepo)
+	assert.True(t, *scan.FailOnSecurityIssues)
```

Comment:
Can we add here verify aggregate == false on default values?

Reply:
added